init/configure-xdebug.sh

```diff
@@ -3,7 +3,7 @@
 # Enable XDebug if XDEBUG_CONFIG environment variable exists
 # XDebug will automatically read config from XDEBUG_CONFIG, so all we need to do here is enable it.
 
-if [ ! -z "$XDEBUG_CONFIG" ]
+if [ -n "$XDEBUG_CONFIG" ]
```

Comment:
Modified for conciseness.
This line produced a double-negative. Using `-n` does the same operation.
More information can be found on [Shellcheck](https://github.com/koalaman/shellcheck/wiki/SC2236)